fecfile_validate_python/LICENSE

```diff
@@ -0,0 +1 @@
+This project is covered by the FEC's https://github.com/fecgov/FEC/blob/master/LICENSE.md.
```

Comment:
Should this be a `.md` file? Also, can the FEC's license be a hyperlink?

Reply:
I based it off of this documentation from PyPa: https://packaging.python.org/en/latest/tutorials/packaging-projects/
I can't find and example of the package license file supporting markdown. pytest and flake8 just have raw text. 
https://github.com/pytest-dev/pytest/blob/main/LICENSE
https://github.com/PyCQA/flake8/blob/main/LICENSE